### PR TITLE
Bug 1764984: storageclusterinit: Update secret name for StorageClasses

### PR DESCRIPTION
**Pull request by kshlm**
The names of the secrets used by StorageClasses for the Rook Ceph CSI drivers
have been updated to follow the changes in Rook

Backport of #230 . Also backports #232 and #205 for CI fixes.

### CODE REVIEW

File: deploy/olm-catalog/ocs-operator/0.0.1/ocs-operator.v0.0.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -929,7 +929,7 @@ spec:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.namespace
-                image: rook/ceph:v1.1.2
+                image: rook/ceph:master
```
Comment (obnoxxx) [inline deploy/olm-catalog/ocs-operator/0.0.1/ocs-operator.v0.0.1.clusterserviceversion.yaml]: We should not be doing master for the release branch.

File: deploy/olm-catalog/ocs-operator/0.0.1/ocs-operator.v0.0.1.clusterserviceversion.yaml [hunk 2]
```diff
@@ -1014,7 +1014,7 @@ spec:
                 - name: OPERATOR_NAME
                   value: ocs-operator
                 - name: CEPH_IMAGE
-                  value: ceph/ceph:v14.2.4-20190917
+                  value: ceph/ceph:v14.2
```
Comment (obnoxxx) [inline deploy/olm-catalog/ocs-operator/0.0.1/ocs-operator.v0.0.1.clusterserviceversion.yaml]: we should be more specific for the release branch,

**Comment by davidvossel** [comment]
> /test red-hat-storage-ocs-ci-e2e-aws

@kshlm red-hat-storage-ocs-ci-e2e-aws is going to be flaky in release-4.2 unless we backport this pr, #205

**Comment by kshlm** [comment]
> @kshlm red-hat-storage-ocs-ci-e2e-aws is going to be flaky in release-4.2 unless we backport this pr, #205

We should do the backport, if it fixes the flakiness. But it would need a bz opened targeting 'OCS 4.2.0' to allow prow to merge it.

**Comment by davidvossel** [comment]
> We should do the backport, if it fixes the flakiness. But it would need a bz opened targeting 'OCS 4.2.0' to allow prow to merge it.

could we backport it as a part of the bz associated with this PR?

**Comment by kshlm** [comment]
> could we backport it as a part of the bz associated with this PR?

I'm fine with doing this.

**Comment by kshlm** [comment]
And I backported #205 as well.

**Comment by jarrpa** [comment]
You should not include the backport of #205 as part of this PR. Please raise another PR for release-4.2, using the same BZ.

**Comment by obnoxxx** [comment]
holding b/c it is superseded by #243

**Comment by obnoxxx** [comment]
closing because #243 is merged

**Pull request closed without merging**